Eatery Blue/Eatery Blue/Testing/DummyData.swift

```diff
@@ -68,5 +70,120 @@ enum DummyData {
             )
         ]
     )
+    
+    static let terraceMenu = Menu(categories: [
+        MenuCategory(category: "Pho Bar", items: [
+            MenuItem(
+                description: "A pho bowl with your choice of toppings.",
+                healthy: false,
+                name: "Pho Bowl",
+                price: 1025
+            ),
+            MenuItem(
+                description: "A rice bowl with your choice of toppings.",
+                healthy: false,
+                name: "Rice Bowl",
+                price: 1025
+            ),
+        ]),
+        MenuCategory(category: "Salads", items: [
+            MenuItem(
+                description: "A mixed salad with your choice of toppings.",
+                healthy: true,
+                name: "Mixed Salad",
+                price: 1100
+            )
+        ])
+    ])
+
+    static let terrace = Eatery(
+        campusArea: "Central",
```

Comment:
iirc `campusArea` takes on discrete values. You might want to consider turning this into a `static enum`.